.github/workflows/sql-benchmarks.yml

```diff
@@ -9,35 +9,28 @@ on:
 
 jobs:
   bench:
-    # S3 is shared state here, and we want to make sure only one of each job runs at a time
+    # S3 is shared state here, and we want to make sure only one of each job per engine runs at a time.
     concurrency:
-      group: ${{ github.workflow }}-${{ github.ref_name }}-${{matrix.id}}
+      group: ${{ github.workflow }}-${{ github.ref_name }}-${{matrix.id}}-${{matrix.engine}}
       cancel-in-progress: false
     strategy:
       fail-fast: false
       matrix:
+        engine: [datafusion, duckdb]
         # Regarding "include:":
         # https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/running-variations-of-jobs-in-a-workflow#example-adding-configurations
         include:
-          - id: tpch-nvme
-            binary_name: tpch
-            name: TPC-H on NVME
-            local_dir: bench-vortex/data/tpch/1
           - id: clickbench-nvme
             binary_name: clickbench
             name: Clickbench on NVME
             local_dir: bench-vortex/data/clickbench_partitioned
-          - id: tpch-s3
-            binary_name: tpch
-            name: TPC-H on S3
-            local_dir: bench-vortex/data/tpch/1
-            remote_storage: s3://vortex-bench-dev-eu/${{github.ref_name}}/tpch-sf1/
+
     runs-on:
       - runs-on=${{ github.run_id }}
-      - family=c6id.8xlarge
+      - family=c6id.32xlarge
```

Comment:
Will revert this before landing the PR. This to speed up development on the GH action.